core/src/com/group66/game/cannon/Cannon.java

```diff
@@ -30,34 +57,74 @@ else if (angle <= 0)
 			angle = 1;
 	}
 	
+	/**
+	 * Sets the angle of the cannon.
+	 *
+	 * @param angle the new angle
+	 */
 	public void setAngle(float angle) {
 		this.angle = angle;
```

Comment:
We could also give the max and min angle to the constructor and do check in the setAngle functions, might be even nicer?